### PR TITLE
msc-generator 8.5

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/m/msc-generator.rb [hunk 1]
```diff
@@ -38,8 +38,6 @@ class MscGenerator < Formula
   depends_on "tinyxml2"
 
   on_macos do
-    # Check if upstream still requires GNU sed and GNU make 4.3+ in future release.
-    # Issue ref: https://gitlab.com/msc-generator/msc-generator/-/issues/92
```
Comment (chenrui333) [inline Formula/m/msc-generator.rb:42]: even though the issue got closed, still need this for future reference.

**Review by chenrui333 — APPROVED** [review]

File: Formula/m/msc-generator.rb [hunk 2]
```diff
@@ -56,19 +55,6 @@ class MscGenerator < Formula
...
-  # Use upstream MR to fix build on macOS. Remove if merged and in release.
-  # Issue ref: https://gitlab.com/msc-generator/msc-generator/-/issues/90
-  # PR ref: https://gitlab.com/msc-generator/msc-generator/-/merge_requests/154
-  patch do
-    url "https://gitlab.com/msc-generator/msc-generator/-/commit/a4d9ef77f797d33970882b1d7639b872f2ced126.diff"
-    sha256 "0cae8315f0aafdb68ccfe94b15da278cd28355a65e7fe30bfc4475d1d0f117e0"
-  end
-
-  # Backport compatibility with graphviz 9.0
-  # TODO: Remove in next release along with `autoreconf` usage in install
```
Comment (cho-m) [inline Formula/m/msc-generator.rb:68]: This comment is for removing following lines:
https://github.com/Homebrew/homebrew-core/blob/ffd36fa1328b521eea915893b429a26ef730dc40/Formula/m/msc-generator.rb#L89-L91

Comment (chenrui333) [reply Formula/m/msc-generator.rb:68]: good call out

Comment (chenrui333) [reply Formula/m/msc-generator.rb:68]: cc @Porkepix

Comment (cho-m) [reply Formula/m/msc-generator.rb:68]: Those steps are for building from git. We use the official tarball. For autotools-based formula, we prefer to use existing configure script and only regenerate if there is something wrong (e.g. too old). If we ever decide to support HEAD, then we would need to run `autogen`.

**Review by cho-m — APPROVED** [review]

**Pull request merged**